.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/compilerla/conventional-pre-commit
+    rev: v1
+    hooks:
+      - id: conventional-pre-commit
+        stages: [commit-msg]
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: end-of-file-fixer
+      - id: requirements-txt-fixer
+      - id: check-yaml
+        args: ["--unsafe"]
+      - id: check-added-large-files
```

Comment:
Not sure if you planned to add the additional linters etc. in another PR (maybe with the Action workflow)? If so no worries! Let's just make sure to get them in there at some point.

Reply:
Yep, exactly. I plan to add them in when I get Python-related files into the repo.